cmd/libvirt-provider/app/app.go

```diff
@@ -96,7 +97,8 @@ type HTTPServerOptions struct {
 }
 
 type ServersOptions struct {
-	Metrics HTTPServerOptions
+	Metrics        HTTPServerOptions
+	LibvirtConnect HTTPServerOptions
```

Comment:
why libvirtConnect?

Reply:
Initially, it was named libvirtConnect. However, to accommodate future extensibility, I later renamed it to healthCheck. It appears that I missed updating the name in a few places. Thank you for bringing this to my attention. I have now changed it to HealthCheck.